__tests__/library.js

```diff
@@ -11,6 +11,7 @@ describe('Checking for the presence of main APIs', () => {
     'RingCollection',
     'SDFileParser',
     'SSSearcher',
+    'CanonizerUtil',
```

Comment:
You put `JSCanonizerUtil.java` in the "core" folder, so this should be part of `coreAPI`.